routes/api.php

```diff
@@ -16,3 +16,9 @@
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::group(['namespace' => 'Api', 'prefix' => 'v1', 'as' => 'api.v1.'], function () {
+    Route::patch('change_order', 'MenuController@changeOrderMenu')->name('change_order');
```

Comment:
xem phần `Do not use underscores ( _ )` trong link này nhé:
https://restfulapi.net/resource-naming/